core/templates/dev/head/css/oppia.css

```diff
@@ -2988,3 +2988,333 @@ md-card.preview-conversation-skin-supplemental-card {
   color: red;
   margin-bottom: 10px;
 }
+
+/*
+  about / learners / creators pages
+*/
+.clearfix {
+  display: table;
+  content: "";
+  clear: both;
+}
+
+.about-header {
+  background-color: rgb(54, 222, 207);
+  background-image: url("../images/general/icons-bg.png");
+  background-blend-mode: soft-light;
+  color: #fff;
+  text-align: center;
+  padding: 10%;
+}
+
+.about-card {
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.about-content {
+  width: 60%;
+  padding: 0;
+}
+
+.about-extra-container {
+  width: 100%;
+}
+
+.about-extra-content {
+  width: 60%;
+  margin: 0 auto;
+  position: relative;
+}
+
+.about-otter-container {
+  position: absolute;
+  display: inline-block;
+  bottom: -2.3em;
+}
+
+.otter-speech-card {
```

Comment:
Try and scope all the CSS class names to clearly point to the About page (e.g. .oppia-about-...), especially if you're not planning to use them elsewhere on the site. Some of the class names seem more general than I assume they're meant to be, and this could invite confusion. Probably better to just start off with scoping them to the About page and then we can generalize individual ones later as the need arises.